docs/DeveloperGuide.md

```diff
@@ -333,33 +333,20 @@ A successful execution of the `viewSchedule` command is described as follows:
     * Person List display only fits a few Persons at a time.
     
 ## 4.4 ViewGroup Feature
-View Group feature allows the user to be able to view a list of persons who share the same tag.
+This section details how the 'viewGroup' command is implemented. This command allows the user to be able to view a list of persons who share the same tag.
 
 ### Implementation
-`ViewGroupParser`, `ViewGroupCommand` and `IsTagInPersonPredicate` classes are involved in the execution of the `ViewGroup` command.
+`ViewGroupParser` and `ViewGroupCommand` classes are involved in the execution of the `ViewGroup` command.
```

Comment:
I think it should be ViewGroupCommandParser?

Reply:
Yup, amended